### ISSUE TITLE
Add functions documentation

### ISSUE THREAD

**Issue by strk** [linz/linz-bde-schema#26]
Functions in this schema have been found to be the bottleneck of some functions contained in [linz-lds-bde-schema](https://github.com/linz/linz-lds-bde-schema) - having a clear documentation about what those functions are supposed to do (even as comments in the SQL file itself) might help finding another way to implement the same functionality when some form of inlining is needed.

See also https://github.com/linz/linz-lds-bde-schema/issues/19

**Comment by palmerj** [comment]
I agree should documentation is required, but should wait until after the NG release
